Source/Tools/FEXRootFSFetcher/Main.cpp

```diff
@@ -284,6 +285,17 @@ void CheckUnsquashfs() {
   }
   close(fd);
 }
+void CheckZenity() {
+  // Check if zenity exists on the host
+  std::vector<const char*> ExecveArgs = {
```

Comment:
Perhaps use a std::array here, which would get rid of a trivial allocation

Reply:
Converted this over to array, then converted the other copy and paste sections of code using vector, which also converts a couple of vector passing arguments to span.